utilities/decrypt_data/export_postgres_to_excel_chunks.py

```
@@ -0,0 +1,585 @@
+            'results': export_results
+        }
+        
+        logger.info(f"Schema '{schema}' export completed: {successful_exports}/{len(tables)} tables exported successfully")
+        logger.info(f"Schema '{schema}' total rows: DB={total_db_rows}, Excel={total_excel_rows}")
+        
+        if failed_exports:
+            logger.warning(f"Failed to export tables: {', '.join(failed_exports)}")
+        
+        return schema_summary
+    
+    def export_multiple_schemas(self, schemas: List[str], output_dir: str = 'exported_tables', use_chunking: bool = True, chunk_size: int = 10000) -> dict:
```

Ruff (0.12.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)